packages/matter.js/src/log/Format.ts

```diff
@@ -419,6 +420,9 @@ function renderValue(value: unknown, formatter: Formatter, squash: boolean): str
             })
             .join(squash ? "" : " ");
     }
+    if (typeof value === "object") {
+        return formatter.text(Logger.toJSON(value));
```

Comment:
Maybe use serialize() here instead of toJSON?  Serialize is a little more readable and won't crash if there are circular references in the object...  Also maybe special case Date just before this so dates don't get quoted.  Other than that seems fine